src/components/sidebar/splitLinkView/SplitLinkView.tsx

```diff
@@ -0,0 +1,219 @@
+                    linkStartNodeId, linkEndNodeId, linkTransitType,
+                );
+                const node = await NodeService.fetchNode(nodeId);
+                this.setState({
+                    link,
+                    node,
+                });
+                this.props.linkStore!.setLink(link);
+                this.props.linkStore!.setIsLinkGeometryEditable(false);
+                this.props.linkStore!.setNodes([node]);
+                const bounds = L.latLngBounds(link.geometry);
+                bounds.extend(node.coordinates);
+                this.props.mapStore!.setMapBounds(bounds);
+                this.props.networkStore!.hideAllMapLayers();
+            }
+        } catch (e) {
+            this.props.errorStore!.addError(
+                    // tslint:disable-next-line:max-line-length
+                    `Haku löytää linkin ja solmun joka olet valinnut epäonnistui`,
+                    e,
+                );
+        }
+        this.setState({ isLoading: false });
+    }
+
+    componentDidMount() {
+        this.init();
+    }
+
+    fetchRoutePaths = async (date: Date) => {
+        this.setState({
+            isLoadingRoutePaths: true,
+        });
+        const link = this.state.link;
+        const routePaths = await RoutePathService.fetchRoutePathsUsingLinkFromDate(
+            link!.startNode.id,
+            link!.endNode.id,
+            link!.transitType!,
+            date,
+        );
+        this.setState({
+            routePaths,
+            isLoadingRoutePaths: false,
+        });
+    }
+
+    updateSelectedDate = (date: Date) => {
+        this.setState({ selectedDate: date });
+        this.fetchRoutePaths(date);
+    }
+
+    toggleIsRoutePathSelected = (routePathId: string) => {
+        const selectedRoutePathIds = this.state.selectedRoutePathIds;
+        let newSelectedRouteIds = [];
+        if (this.state.selectedRoutePathIds.includes(routePathId)) {
+            newSelectedRouteIds = selectedRoutePathIds.filter(id => id !== routePathId);
+        } else {
+            selectedRoutePathIds.push(routePathId);
+            newSelectedRouteIds = selectedRoutePathIds;
+        }
+        this.setState({ selectedRoutePathIds: newSelectedRouteIds });
+    }
+
+    saveSplittedLink = () => {
+        const splittedRoutePaths =
+            this.state.routePaths
+                .filter(rp => this.state.selectedRoutePathIds.includes(rp.internalId));
+        // tslint:disable-next-line
+        console.log(splittedRoutePaths);
+        this.props.alertStore!.setFadeMessage('Linkin jako vielä kehitetään', AlertType.Info);
+    }
+
+    selectAllRoutePaths = () => {
+        this.setState({
+            selectedRoutePathIds: this.state.routePaths.map(rp => rp.internalId),
+        });
+    }
+
+    clearSelectedRoutePaths = () => {
+        this.setState({
+            selectedRoutePathIds: [],
+        });
+    }
+
+    render() {
+        const isSaveButtonDisabled = false;
```

Comment:
Save button should sometimes be disabled

Reply:
Hmm, should it?

Reply:
Its now disabled if we have date selected, but no routepaths selected